test/functional/multiwallet.py

```diff
@@ -26,13 +26,34 @@ def run_test(self):
         self.assert_start_raises_init_error(0, ['-wallet=w1', '-wallet=w1'], 'Error loading wallet w1. Duplicate -wallet filename specified.')
 
         # should not initialize if wallet file is a directory
-        os.mkdir(os.path.join(self.options.tmpdir, 'node0', 'regtest', 'w11'))
+        os.mkdir(os.path.join(self.options.tmpdir, 'node0', 'regtest', 'wallets', 'w11'))
```

Comment:
Perhaps make some local variables here rather than repeating the same calls to `os.path.join()`:

```python
wallet_dir = os.path.join(self.options.tmpdir, 'node0', 'regtest', 'wallets')
...
wallet_dir2 = os.path.join(self.options.tmpdir, 'node0', 'regtest', 'walletdir')
```

or whatever